app/main.py

```
@@ -1,9 +1,14 @@
 from fastapi import FastAPI
-from app.routers import health_check
+from app.routers import health_check, file_downloader, file_streamer
+from app.routers import analyze, health_check
```

Ruff (0.8.2)

Redefinition of unused `health_check` from line 2

Remove definition: `health_check`

(F811)